tests/CreateTest.php

```diff
@@ -249,6 +249,19 @@ public function services_can_be_marked_as_private()
     /**
      * @test
      */
+    public function services_can_be_marked_as_unshared()
+    {
+        $container = new ContainerBuilder;
+        (new PhpConfigLoader($container))->load([
+            'bar' => create('stdClass')
+                ->unshared()
+        ]);
+        self::assertFalse($container->findDefinition('bar')->isShared());
```

Comment:
Could you rather write a high level test? E.g. test that when you retrieve the service twice it is not the same instance.

That way the test is less coupled to the internals of the container.